frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -2081,12 +2097,19 @@ mod tests {
 				vec![
 					Event::SignedPhaseStarted { round: 1 },
 					Event::UnsignedPhaseStarted { round: 1 },
-					Event::ElectionFinalized { election_compute: None },
+					Event::ElectionFinalized { election_compute: None, score: None },
 					Event::SolutionStored {
 						election_compute: ElectionCompute::Fallback,
 						prev_ejected: false
 					},
-					Event::ElectionFinalized { election_compute: Some(ElectionCompute::Fallback) }
+					Event::ElectionFinalized {
```

Comment:
Please add these event checks to at least one test that does `ElectionCompute::Signed` and one that does `ElectionCompute::Unsigned` and make sure the score is sane.

Reply:
ping.